mono/metadata/gc.c

```diff
@@ -564,6 +564,17 @@ ves_icall_System_GC_KeepAlive (MonoObjectHandle obj, MonoError *error)
 	 */
 }
 
+void
+ves_icall_System_GC_GetGCMemoryInfo (void* self,
```

Comment:
This `void* self` thing should be worked out. Other icalls do not have it.

Reply:
It is the space for the by-value returned strut, which isn't used.
You meant it to return void but misdeclared in managed.

Reply:
Got it.